### PR TITLE
Feat UI: Table Overflow Issue in Data Section

### LINKED ISSUE #28: BUG: Issues with the UI
**Describe the bug**
1. The table under the **Data** section exceeds the panel width, unlike the table in the **Algorithms**/**Leaderboard** sections, which are properly contained.  
2. The following **Page Tour** button is not functioning.
Image

**Expected behavior**
1. All the tables should be within the panel width.
2. Either we fix it or exclude it for now.

Responsible: @Rakesh-Seenu 
Accountable: @gurdeep330

### PR DESCRIPTION
**Pull request by Rakesh-Seenu**
# For authors
Before:
image

After:
image

## Description
This pull request removes the "Click here for a quick intro of the page" button from the Data section for now, as it was not functioning correctly and causing UI clutter. Previously, the button was displayed as shown in the uploaded screenshot. Now, it has been removed to streamline the user experience.

Changes Made:

- Removed the "Intro Page" button in the Data section.
- Adjusted the layout of the page to ensure consistency without the intro button.

Please refer to the screenshot above for the issue before and after the fix:

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Please describe the tests you conducted to verify your changes. These may involve creating new test scripts or updating existing ones.
- [ ] Added new test(s) in the ```tests``` folder
- [ ] Added new function(s) to an existing test(s) (e.g.: ```tests/testX.py```)
- [x] No new tests added (Please explain the rationale in this case)

## Checklist
- [x] My code follows the style guidelines mentioned in the Code/DevOps guides
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. MkDocs)
- [ ] My changes generate no new warnings
- [ ] I have added or updated tests (in the ```tests``` folder) that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# For reviewers

## Checklist pre-approval
- [ ] Is there enough documentation?  
- [ ] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
- [ ] Does the test(s) successfully test edge/corner cases?
- [ ] Does the PR pass the tests? (if the repository has continuous integration)

## Checklist post-approval
- [ ] Does this PR merge ```develop``` into ```main```? If so, please make sure to add a prefix (feat/fix/chore) and/or a suffix BREAKING CHANGE (if it's a major release) to your commit message.
- [ ] Does this PR close an issue? If so, please make sure to descriptively close this issue when the PR is merged.

## Checklist post-merge
- [ ] When you approve of the PR, merge and close it (Read this [article](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github) to know about different merge methods on GitHub)
- [ ] Did this PR merge ```develop``` into ```main``` and is it suppose to run an automated release workflow (if applicable)? If so, please make sure to check under the "Actions" tab to see if the workflow has been initiated, and return later to verify that it has completed successfully.

### CODE REVIEW

File: image.png
Comment (gurdeep330) [inline image.png:1]: This has changed, right?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # **THIS REPOSITORY IS A WORK IN PROGRESS!**
 # VPELeaderboard
 
-![Test Status](https://img.shields.io/badge/TESTS%20VPE%20Leaderboard-passing-brightgreen)
-![Release](https://img.shields.io/badge/release-v1.0.0-blue)
+[![Test Status](https://img.shields.io/badge/TESTS%20VPE%20Leaderboard-passing-brightgreen)](https://github.com/VirtualPatientEngine/VPELeaderboard/actions/workflows/tests.yml)
+![GitHub Release](https://img.shields.io/github/v/release/VirtualPatientEngine/VPELeaderbord)
```
Comment (gurdeep330) [inline README.md:5]: This link does not work.

File: docs/algorithms/index.md
Comment (gurdeep330) [inline docs/algorithms/index.md:1]: Aren't all the .md files in the docs folder getting updated by themselves when the mkdocs workflow runs? In that case, you don't need to commit the .md files...

**Review by gurdeep330 — CHANGES_REQUESTED** [review]
@Rakesh-Seenu 

Thanks for updating. I have a few minor comments regarding the typos and conflicts with the main branch.

File: docs/algorithms/index.md
Comment (Rakesh-Seenu) [reply docs/algorithms/index.md:1]: yes, it is updated when the workflow runs.

File: image.png
Comment (gurdeep330) [reply image.png:1]: No, but I still see the old one:
image


And, if I remember correctly, you had moved it to the docs folder?

File: docs/algorithms/index.md
Comment (gurdeep330) [reply docs/algorithms/index.md:1]: If that is the case, I'd recommend not committing these files to your PR branch in the first place.

File: image.png
Comment (Rakesh-Seenu) [reply image.png:1]: yes, i have moved it to docs folder and now the image is as below
image

File: docs/algorithms/index.md
Comment (Rakesh-Seenu) [reply docs/algorithms/index.md:1]: I have updated the `.gitignore` file to ensure that the generated Markdown files are not committed to the repository. These files are generated and saved automatically during the workflow execution.

File: image.png
[the remark this answers is not in the document]
Comment (Rakesh-Seenu) [reply image.png:1]: I am sorry there was another image in the root, I have removed it.
